Installers/Windows/MonoGame.nsi

```diff
@@ -106,6 +106,11 @@ Section "MonoGame Core Components" CoreComponents ;No components page, name is n
   File '..\..\MonoGame.Framework\bin\Android\AnyCPU\Release\*.dll'
   File '..\..\MonoGame.Framework\bin\Android\AnyCPU\Release\*.xml'
   
+  ; Install OUYA Assemblies
```

Comment:
Something's wrong with this branch, there are 10 references to the OUYA platform.